readme.md

```diff
@@ -23,13 +23,13 @@ This is possible because [we use the Parcel library](https://en.parceljs.org/scs
 
 ❗️ Replace `<your_account>` with your GitHub username and copy the links to the `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://disiol.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://disiol.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="942" alt="image" src="https://github.com/user-attachments/assets/3f3bf581-5144-4b37-b973-3c7ecd2f74b7" />